docs/src/main/sphinx/release/release-419.md

```diff
@@ -19,16 +33,33 @@
 
 * Improve performance of reading ORC files with Bloom filter indexes. ({issue}`17530`)
 * Improve performance of reading Parquet files. ({issue}`17612`)
+* Improve optimized Parquet writer performance for
+  [structural data types](structural-data-types). ({issue}`17665`)
+* Fix query failure when reading ORC files with a shorthand timezone id in the
```

Comment:
This is confusing. It adds new behavior to support legacy shorthand timezones, but it's listed as "Fix query failure". I would rephrase as "Add support for reading ORC files with shorthand timezone ids in the stripe footer metadata".